AnswerKingAPI.yml

```diff
@@ -2,15 +2,230 @@ openapi: 3.0.1
 info:
   title: Answer King API
   version: v1
+tags:
+  - name: Inventory
+    description: Manage the inventory.
+  - name: Orders
+    description: Create and manage customer orders.
+  - name: Tags
+    description: Create and manage tags.
 paths:
+  /api/tags:
+    post:
+      tags:
+        - Tags
+      summary: Create Tag.
+      operationId: craeteTag
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Tag.Request'
+      responses:
+        '201':
+          description: Tag created.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Tag.Response'
+        '400':
+          description: Invalid parameters are provided.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ProblemDetails'
+    get:
+      tags:
+        - Tags
+      summary:  List All tags.
```

Comment:
is there a reason for the double-spaces here? they're showing up in a few places

Reply:
no reason, will be fixed.